2-Browsers/Week1/homework/ex4-whatsTheTime/index.js

```diff
@@ -7,6 +7,60 @@
 ------------------------------------------------------------------------------*/
 function addCurrentTime() {
   // TODO complete this function
+  const date = new Date();
+  let h = date.getHours();
+  let m = date.getMinutes();
+  let s = date.getSeconds();
+  let session = 'AM';
+
+  if (h === 0) {
+    h = 12;
+  }
+
+  if (h > 12) {
+    h = h - 12;
+    session = 'PM';
+  }
+
+  h = h < 10 ? '0' + h : h;
```

Comment:
Could you refactor these 3 lines to remove the duplicate code?

Reply:
I did.😊